code/datums/components/tackle.dm

```diff
@@ -163,24 +163,24 @@
 		if(-4 to -2) // glancing blow at best
 			user.visible_message("<span class='warning'>[user] lands a weak [tackle_word] on [target], briefly knocking [target.p_them()] off-balance!</span>", "<span class='userdanger'>You land a weak [tackle_word] on [target], briefly knocking [target.p_them()] off-balance!</span>", target)
 			to_chat(target, "<span class='userdanger'>[user] lands a weak [tackle_word] on you, briefly knocking you off-balance!</span>")
-
 			user.Knockdown(30)
+			target.adjustStaminaLoss(30)
 			target.apply_status_effect(STATUS_EFFECT_TASED_WEAK, 6 SECONDS)
 
 		if(-1 to 0) // decent hit, both parties are about equally inconvenienced
 			user.visible_message("<span class='warning'>[user] lands a passable [tackle_word] on [target], sending them both tumbling!</span>", "<span class='userdanger'>You land a passable [tackle_word] on [target], sending you both tumbling!</span>", target)
 			to_chat(target, "<span class='userdanger'>[user] lands a passable [tackle_word] on you, sending you both tumbling!</span>")
 
-			target.adjustStaminaLoss(stamina_cost)
+			target.adjustStaminaLoss(stamina_cost * 1.5)
 			target.Paralyze(5)
 			user.Knockdown(20)
-			target.Knockdown(25)
+			target.Knockdown(30)
```

Comment:
hard knockdowns are kinda shitty

Reply:
Replace knockdowns with standard knockdowns? Do they still do the thing where you cant get up instantly and run off?